book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,261 @@
+    *   You can get a feel for it in the [demo](https://openbis-eln-lims.ethz.ch/openbis/webapp/eln-lims/) deployment.
+    *   openBIS is a bit more complex to administer [based on its documentation](https://openbis.ch/index.php/docs/admin-documentation/).
+    *   As openBIS is developed at ETH Zurich it can be hosted for you under the openRDM service operated by ETH Zurich scientific IT services.
+        No fixed pricing is available - cost would be dependent on your specific needs.
+    
+* [OSF](https://osf.io/)
+    *   OSF is oriented towards sharing and collaborating on your work, including the ability to generate DOIs and host pre-prints directly on the main instance.
+    *   It is free to use OSF at the main instance at [osf.io](https://osf.io/).
+        For larger data you must provide your own additional [storage add-ons, available from a number of cloud storage providers](https://help.osf.io/article/395-storage-add-ons).
+    *   Whilst you could deploy your own self-hosted OSF this is not how it is intended to be used (except for development).
+    *   The OSF has strong sharing features, making it easy to share parts of your ELN publicly.
+
+When using an open solution it is always important to consider how you can contribute to its ongoing development and maintenance, for example by donating to the project.
+If it makes sense given the makeup of your organisation and that of the project you may also be able to contribute developer time to the project.
+
+(rr-rdm-using-non-elns-for-eln-things)
+## Using non-ELNs for ELN things
+
+Note taking and personal knowledge management (PKM) applications of various kinds have been pushed into service as ELNs by many individuals and institutions, despite not being designed specifically for this tasks.
+Some tools are better suited to be used 'off-label' as ELNs than others.
+Some may have significant drawbacks compared to purpose built solutions, despite being better at some things than currently available dedicated ELN options.
+
+Microsoft OneNote is quite popular as an ELN despite a proprietary notebook format with limited export options and that [microsoft has entertained getting rid of the good (desktop) version this of application in favour of its much less featureful 'web' version in the past. Though they have apparently relented, for now.](https://www.youtube.com/watch?v=4IkUfh05TGE).
+OneNote is a good experience as a free-form note taking tool especially for hand-written or drawn notes and automatic linking of anotations to audio timestamps.
+The lack of a good version history is a distinct downside.
+Its popularity likely stems, in part, from its availability in the Microsoft office suite.
+Software which most institutions will already have, meaning they will not incurr additional costs if they use OneNote.
```

Comment:
```suggestion
```
I would combine these two sentences, otherwise it is not very clear what you mean by the current one.